x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -190,6 +191,59 @@ export const hasTimestampFields = async (args: {
   return { foundNoIndices: false, warningMessage: undefined };
 };
 
+export const checkForFrozenIndices = async ({
```

Comment:
I think a quick JSDoc here would be helpful: one might guess that the `string[]` being returned is a list of frozen indices, but better to have a comment confirming that.

Reply:
Added a JSDoc for this function.